tests/playbooks/e2e.yaml

```diff
@@ -22,6 +24,50 @@
             url: "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
         wait: yes
 
+    - name: Create a clone of previous PVC
+      kubevirt_pvc:
+        state: present
+        name: "{{ pvc2_name }}"
+        access_modes:
+        - ReadWriteOnce
+        size: 100Mi
+        cdi_source:
+          pvc:
+            namespace: default
+            name: "{{ pvc_name }}"
+        wait: yes
+
+    - name: Create an ephemeral VM with the cloned PVC
+      kubevirt_vm:
+        state: present
+        name: "{{ vmi_name }}"
+        ephemeral: yes
+        memory: 64Mi
+        cpu_cores: 1
+        disks:
+        - name: rootdisk
+          volume:
+            persistentVolumeClaim:
+              claimName: "{{ pvc2_name }}"
+          disk:
+            bus: virtio
+
+    - name: Verify the ephemeral VM is in fact running
+      k8s_facts:
+        name: "{{ vmi_name }}"
+        kind: VirtualMachineInstance
+        api_version: kubevirt.io/v1alpha3
+      register: vmi
+      until: "vmi.resources[0].status['phase'] == 'Running'"
+      delay: 1
```

Comment:
Why do we need this so low? Do we assume that vm is always there if so why do we need it here?

Reply:
Because this isn't supposed to be looped wait logic anymore, as the wait logic is inside  kubevirt_vm now. This is supposed to be more of an assert verifying that the wait logic inside kubevirt_vm worked correctly and the VMI is really ready (ditto for the rest of similar instances).

This is just WIP, so I wanted to "annotate" all the relevant parts of the playbooks for myself and this was the quickest way to do it (near instant loop with little to no repeats == almost an assert). I'll likely replace this non–loopy loop with a proper k8s_facts and an assert at some later time.